lib/TH/generic/THTensorMath.c

```diff
@@ -1066,6 +1066,30 @@ void THTensor_(cross)(THTensor *r_, THTensor *a, THTensor *b, int dimension)
                        r__data[2*r__stride] = a_data[0*a_stride]*b_data[1*b_stride] - a_data[1*a_stride]*b_data[0*b_stride];);
 }
 
+void THTensor_(cmax)(THTensor *r, THTensor *t, THTensor *src) {
+  THTensor_(resizeAs)(r, t);
+  TH_TENSOR_APPLY3(real, r, real, t, real, src,
```

Comment:
This code is consistent with the rest of Torch (see for example the implementations of `minall` and `maxall`). If we wanted to change it, it should be changed everywhere.